pages/about.css

```diff
@@ -0,0 +1,70 @@
+
+.SCA-about {
+
+}
+
+.titleschool {
```

Comment:
You should remove this since we put it in index.css

Reply:
ok